src/main/java/com/artipie/scripting/ScriptRunner.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.scripting;
+
+import com.artipie.ArtipieException;
+import com.artipie.asto.Key;
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.settings.Settings;
+import com.artipie.settings.repo.RepositoriesFromStorage;
+import com.jcabi.log.Logger;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import javax.script.ScriptException;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.quartz.JobKey;
+import org.quartz.SchedulerException;
+import org.quartz.impl.StdSchedulerFactory;
+
+/**
+ * Script runner.
+ * Job for running script in quartz
+ *
+ * @since 0.30
+ */
+public final class ScriptRunner implements Job {
+    @Override
+    public void execute(final JobExecutionContext context) throws JobExecutionException {
+        final Settings settings = (Settings) context.getJobDetail().getJobDataMap().get("settings");
+        final Key key = new Key.From(context.getJobDetail().getJobDataMap().getString("key"));
+        final BlockingStorage storage = new BlockingStorage(settings.configStorage());
+        if (settings == null || key == null || storage == null) {
```

Comment:
Condition 'settings == null || key == null || storage == null' is always 'false'